blazor/dropdown-list/style.md

```diff
@@ -56,54 +112,20 @@ Use the following CSS to customize the focusing color of outline theme.
 }
 ```
 
-## Customizing the disabled component’s text color
-
-Use the following CSS to customize the text color when the component is disabled.
-
-```css
-.e-input-group.e-control-wrapper .e-input[disabled] {
-    -webkit-text-fill-color: #0d9133;
-}
-```
+You can use `e-outline` to the [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property to achieve outline theme
```

Comment:
Use the `e-outline` to the [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property to achieve the outline theme.